preprocessing/test_train.py

```diff
@@ -0,0 +1,83 @@
+import csv
```

Comment:
Can we rename this to something else, like `train_test_split.py` or something that at least has the word `split`.